Python_Scripts/loadModels.py

```diff
@@ -19,3 +25,45 @@ def load_specific_model(model_name, explicit_model):
     model = model_module.getModel()
 
     return model
+
+def get_submodel(submodel_path):
+    # load the amici model
+    # add model path
+    amici_model_path = os.path.join(DIR_MODELS, submodel_path)
+    sys.path.insert(0, os.path.abspath(amici_model_path))
```

Comment:
potentially prepend by `if stuff not in sys.path`, to avoid an overly long search PATH

Reply:
done